websites/A/AnimeVost/metadata.json

```
@@ -0,0 +1,66 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "296716315547402240",
+		"name": "ZebroUnion#2632"
+	},
+	"service": "AnimeVost",
+	"altnames": [
+		"АнимеВост"
+	],
+	"description": {
+		"en": "AnimeVost - Base №1 to watch anime online for free on Russian.",
```

Grammar wise this is better.
```suggestion
		"en": "AnimeVost - Best website to watch anime online for free in Russian.",
```